api/workload/v2/workload.go

```diff
@@ -0,0 +1,177 @@
+package workload
+
+import (
+	"context"
+	"crypto"
+	"crypto/tls"
+	"crypto/x509"
+	"sync"
+
+	"github.com/spiffe/spire/proto/api/workload"
+	"go.uber.org/zap"
+)
+
+const (
+	// DefaultAgentAddress is the default GRPC address to contact the spire agent at.
+	DefaultAgentAddress = "unix:///tmp/agent.sock"
+)
+
+// X509SVIDs is an X.509 SVID response from the SPIFFE Workload API.
+type X509SVIDs struct {
```

Comment:
yeah it might be nice to have things like `Default()` and possibly `GetForKeyAgreement()` and `GetForDigitalSignatures()` and whatever else comes out of the discussions that are occurring right now around usage of multiple SVIDs

Reply:
The workload API also includes a CRL and map of federated bundles (https://github.com/spiffe/spire/blob/master/proto/api/workload/workload.proto#L10). I think I made this a struct to allow for it to be extended to include those things.. perhaps we should just add them now?

Reply:
Lets add them, I would say you can defer actually supporting them until we push this PR upstream. Having the placeholders there in the short term seems reasonable.

Reply:
+1